Makefile

```diff
@@ -10,8 +10,8 @@ run:
 		speech-proxy:build
 
 test:
-	curl -i -X POST http://localhost:9001/ -H 'Content-Type: audio/opus' --data-binary @speech_orig.opus
-	curl -i -X POST http://localhost:9001/ -H 'Content-Type: audio/webm' --data-binary @webm.webm
+	curl -i -X POST http://localhost:9001/ -H 'Content-Type: audio/opus' -H 'Accept-Language: en-us' -H 'Store-Sample: 1' -H 'Store-Transcription: 1' -H 'Product-Tag: shell-curl' --data-binary @speech_orig.opus
+	curl -i -X POST http://localhost:9001/ -H 'Content-Type: audio/webm' -H 'Accept-Language: en-us' -H 'Store-Sample: 1' -H 'Store-Transcription: 1' -H 'Product-Tag: shell-curl' --data-binary @webm.webm
```

Comment:
nit: it'd be good to test with different metadata options too:

* success with store-sample 0
* success without store-sample
* success with store-transcription 0
* success without store-transcription
* failure for invalid accept-language
* failure for invalid product-tag
* failure for invalid store-sample
* failure for invalid store-transcription

I think we can test failure (at least that it returns the right status code) as follows:

```
curl -s -o /dev/null -w "%{http_code}" -i -X POST http://localhost:9001/ -H 'Content-Type: audio/webm' -H 'Accept-Language: \0\0' | grep 400 
```